File: Data Structures & Algorithms/duplicate-integer/submission-1.cpp
#include <unordered_set>
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        unordered_set<int> my_set;
        for(int i:nums){
            if(my_set.count(i)){
                return true;
            }else{
                my_set.insert(i);
            }
        }
        return false;
    }
};

File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for k in range(len(nums)):
            for j in range(k+1,len(nums)):
                if nums[k]+nums[j]==target:
                    return [k,j]
                j+=1
            k+=1

            
        